GSA_Adapter/AdapterActions/Execute.cs

```diff
@@ -31,114 +34,52 @@ public partial class GSAAdapter
         /**** IAdapter Interface                        ****/
         /***************************************************/
 
-        public override bool Execute(string command, Dictionary<string, object> parameters = null, Dictionary<string, object> config = null)
+        public override Output<List<object>, bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
```

Comment:
Think this looks great in general!

Some missing commands:
- `NewModel`
- `Analyse`

Also need a fallback method accepting `IExecuteCommand` raising warnings that the command is not supported.

Reply:
Done!